modules/virt-creating-secondary-udn-namespace.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-secondary-udn.adoc              
+
+:_mod-docs-content-type: PROCEDURE                                    
+[id="virt-creating-secondary-udn-namespace_{context}"]                                   
+= Creating a namespace for secondary user-defined networks by using the CLI
+
+You can create a namespace to be used with an existing secondary cluster-scoped user-defined network (CUDN) by using the CLI.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` permissions.
+* You have installed the OpenShift CLI (`oc`).
+
+
+.Procedure
+. Create a `Namespace` object as a YAML file similar to the following example:
```

Comment:
```suggestion
. Create a `Namespace` object similar to the following example:
```
I don't think Kubernetes objects are defined other ways in these cases, so you can skip mentioning YAML here